docs/UserGuide.adoc

```diff
@@ -35,26 +35,48 @@ to get started.
 
 Below is a list of things to do to use the application:
 
-.  Ensure you have Java `11` or above installed in your computer. +
+1.  Ensure you have Java `11` or above installed in your computer. +
 If not, install it link:{javaURL}[here].
-.  Download the latest `organice.jar` link:{repoURL}/releases[here].
-.  Copy the jar file to the folder you want to use as the home folder for your application.
-.  Double-click the file to start the application. The application should appear in a few seconds.
-The following screenshot is a sample of how the application looks like:
-+
+2.  Download the latest `organice.jar` link:{repoURL}/releases[here].
+3.  Copy the jar file to the folder you want to use as the home folder for your application.
+4.  Double-click the file to start the application. The application should appear in a few seconds.
```

Comment:
Double-click the JAR file may sounds a bit easier to understand for slow user

Reply:
Amended.